superset/common/query_context_processor.py

```diff
@@ -87,12 +87,10 @@ class QueryContextProcessor:
     to retrieve the data payload for a given viz.
     """
 
+    AGGREGATED_JOIN_COLUMN = "$aggregated_join_column"
```

Comment:
In general we've assumed that double underscores mean Superset internal columns. For consistency, do you think it could be possible to call this "__aggregated_join_column"?